### PR TITLE
tweaks on credential file, install-local

### PR DESCRIPTION
**Pull request by dbrinegar**
Cinnamon documents this file with a dot, not an underbar. This changes
to be consistent with that file name.

Also changes `install-local` to install from the folder contents instead
of a tarball, as what you want to install are your local file changes
and not some old tarball by mistake.  This also eliminates the constant
rewriting of a previous release.

### CODE REVIEW

**Comment by dbrinegar** [comment]
didn't realize `install-local` was critical path to the travis test, prolly need to make a separate target

File: enterprise-suite/install-es.sh [hunk 1]
```diff
@@ -64,7 +64,7 @@ function import_credentials() {
 # User overridable variables.
```
Comment (marcoderama) [inline enterprise-suite/install-es.sh]: While we're at it, does it make sense to move this to `enterprise-suite/scripts`?

Comment (dbrinegar) [reply enterprise-suite/install-es.sh]: yeah good question.  thoughts @jsravn ?

Comment (jsravn) [reply enterprise-suite/install-es.sh]: Customers will be using this shell script, that's why I left it here. But we could put it in `scripts`, it's not a big deal I think.

Comment (jsravn) [reply enterprise-suite/install-es.sh]: Did you end up moving it @dbrinegar?

Comment (marcoderama) [reply enterprise-suite/install-es.sh]: He did not.  I could move it as part of https://github.com/lightbend/helm-charts/pull/100...

Comment (marcoderama) [reply enterprise-suite/install-es.sh]: Created https://github.com/lightbend/helm-charts/pull/105 for moving the `install-es.sh` script.

**Pull request merged**